.node-version

```diff
@@ -1 +1 @@
-5.2.0
+6.11.0
```

Comment:
you should also update the version of node [on travis](https://github.com/vinay-pad/sfmovies/blob/45669df73fe0c0b9dca00099089d42375ec1aaf8/.travis.yml#L4) to be the exact same as the one you use locally